website/src/components/MiniBlog/styles.ts

```diff
@@ -148,6 +165,14 @@ const useStyles = makeStyles((theme: Theme) => ({
     marginRight: theme.spacing(1),
     cursor: "pointer",
   },
+  sliderWrapper: {
+    [theme.breakpoints.down("sm")]: {
+      padding: theme.spacing(0, 1),
+      '& .slick-list': {
+        padding: `${theme.spacing(0, 3,0 ,0)} !important`
```

Comment:
cannot we avoid `!important`

Reply:
no the this added to override the inline style added by the script